### DISCUSSION TITLE
Email notification after an event is added

### DISCUSSION THREAD

**Discussion by webmark487** [liedekef/events-made-easy#664] [General]
Hi,

How can I achieve, that I get an email notification after an event is added?

Is that possible with some configuration? I didn´t find any.

So I referred to https://www.e-dynamics.be/wordpress/category/documentation/12-hooks-and-filters/

and inserted into the themes functions.php the following code:

add_action('eme_insert_event_action','eme_mail_event');
function eme_mail_event ($event) {
$contact = eme_get_event_contact ($event);
$contact_email = $contact->user_email;
$contact_name = $contact->display_name;
$subject_format="This is the new event called ' #_EVENTNAME '";
$body_format="This is the new event called ' #_EVENTNAME '";

$subject=eme_replace_placeholders($subject_format, $event, "text");
$body=eme_replace_placeholders($body_format, $event, "text");
$blogusers = get_users();
foreach ( $blogusers as $user ) {
eme_send_mail($subject,$body, $user->user_email, $user->display_name, $contact_email, $contact_name);
}
}
but unfortunately I don´t receive an email …

May the theme be the problem? Does the theme have to provide something?

Best regards and thank you,
Markus

**Comment by liedekef** [comment]
Does EME mail work in normal cases? Also: the value for $contact_email in your example is being used as from and reply-to, so you need to make sure your mailserver allows this as a From-address.
Btw, I updated the EME example a bit to call eme_replace_event_placeholders (eme_replace_placeholders is an alias for this function) and eme_queue_mail (the last function is preferred over eme_send_mail, but the function arguments are in a different order)

**Comment by liedekef** [comment]
Also: what EME version are you using?

**Comment by liedekef** [comment]
no answer, closing discussion.

**Comment by webmark487** [comment]
sorry, why do you close this discussion?

**Reply by liedekef** [reply]
I asked for extra information 4 days ago and got no feedback.

**Comment by webmark487** [comment]
should I open another thread with this issue or can it be opened once more?

**Reply by liedekef** [reply]
it's reopened

**Reply by liedekef** [reply]
Btw: if you're not providing the requested info, this will close tomorrow again.

**Comment by webmark487** [comment]
EME Version 2.5.4
Events Made Easy Frontend Submit Version 1.0.87

**Comment by webmark487** [comment]
I can send  testemails with EME, it first says "The e-mail has been sent.", and then I receive the testemail.
I use php mailfunction as the sending method for e-mails

**Reply by liedekef** [reply]
Your EME version is way too old, we're currently at 2.5.14, please update first.
And the frontend submit plugin has been integrated into EME itself, no longer needed. See https://www.e-dynamics.be/wordpress/frontend-submit/

**Reply by liedekef** [reply]
Also, you"re talking about the frontend submit plugin. For events triggered with the new shortcode [eme_add_event_form] (or the old plugin for that matter), the action hook eme_insert_event_action is not executed, but another one is. For the shortcode [eme_add_event_form]  you have the action hook eme_fs_submit_event_action. Next to that, you have a setting "Contact person" that defines the person receiving the alerts concerning new events arriving via frontend submit.

**Comment by webmark487** [comment]
ok, so i will migrate the frontend submit plugin settings now the new eme version in my backend ... and then I will give it another try
